### ISSUE TITLE
Bug in AdaptiveSmoothForm.cs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andrewkirillov/iplab#12]
```
What version of the product are you using?
Ver 2.7.0

Please provide any additional information below.

Current calculation of trackBar value: 
trackBar.Value = (int) factor * 10;

As "Typecasting" has higher priority, "factor" loses its fraction part first 
then gets multiplied by 10.

Correct calculation of trackBar value:
trackBar.Value = (int) (factor * 10);


--
Abhishek Johri
Tokyo
```

Original issue reported on code.google.com by `absolute...@gmail.com` on 21 Sep 2011 at 7:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `andrew.k...@gmail.com` on 21 Sep 2011 at 8:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Corrected Adaptive Smoothing form so it initializes track bar correctly.

Committed in revision 66. Will be released in version 2.8.0.
```

Original comment by `andrew.k...@gmail.com` on 6 Oct 2011 at 8:43
- Changed state: **Fixed**
- Added labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `andrew.k...@gmail.com` on 26 Mar 2012 at 8:48
- Changed state: **Released**
